python_modules/libraries/dagster-aws/dagster_aws_tests/emr_tests/test_pyspark.py

```diff
@@ -124,7 +126,7 @@ def test_pyspark_emr(mock_is_emr_step_complete, mock_read_events):
             "SlaveInstanceType": "c3.xlarge",
         },
         JobFlowRole="EMR_EC2_DefaultRole",
-        LogUri="s3://mybucket/log",
+        LogUri="s3://my/log",
```

Comment:
nit: "s3://{bucket}/log".format(bucket=mock_s3_bucket.name)?

Reply:
Oh yeah, looks like there are actually no assertions that depend on this being set correctly so my errant find/replace didn't cause issues 😐 .